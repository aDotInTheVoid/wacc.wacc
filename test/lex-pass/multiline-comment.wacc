a #foo
b 
# bar
c 
# baz
#quz
d 

##
###
#####

#
##

e