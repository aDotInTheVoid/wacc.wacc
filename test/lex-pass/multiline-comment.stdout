1 identifier a
1 identifier b
1 identifier c
1 identifier d
1 identifier e
0 eof 
